vllm/model_executor/models/qwen2_vl.py

```diff
@@ -307,6 +307,8 @@ def forward(
         x: torch.Tensor,
         cu_seqlens: torch.Tensor,
         rotary_pos_emb: torch.Tensor,
+        max_seqlen: int = None,
+        seqlens: List = None,
```

Comment:
```suggestion
        seqlens: Optional[list[int]] = None,
```
We should avoid using deprecated python 3.8 typing.

Reply:
Thank you for your suggestion